lib/_tls_wrap.js

```diff
@@ -860,6 +861,33 @@ TLSSocket.prototype.renegotiate = function(options, callback) {
   return true;
 };
 
+TLSSocket.prototype.exportKeyingMaterial = function(length, label) {
+  if (typeof length !== 'number') {
+    throw new ERR_INVALID_ARG_TYPE(
+      'length',
+      'number',
+      length
+    );
+  }
+
+  assert(length > 0,
+         'length is not a positive number: ' + length);
+
+  if (typeof label !== 'string') {
+    throw new ERR_INVALID_ARG_TYPE(
+      'label',
+      'string',
+      label
+    );
+  }
```

Comment:
Likewise, you can use `validateString(label, 'label')` here.

Reply:
thanks, wasn't aware of the validators. refactored it